spec/features/admin/deactivating_users_spec.rb

```diff
@@ -1,12 +1,16 @@
 require 'rails_helper'
 
 feature 'deactivating users' do
-  given!(:team_admin)        { find_or_create :team_admin }
-  given!(:manager)        { find_or_create :disclosure_bmt_user }
-  given!(:responder)      { create :responder, responding_teams: [bu] }
-  given!(:user_with_cases){ find_or_create :foi_responder }
-  given!(:live_case)      { create :accepted_case }
-  given!(:bu)             { find_or_create :foi_responding_team }
+  given!(:team_admin)       { find_or_create :team_admin }
+  given!(:manager)          { find_or_create :disclosure_bmt_user }
+  given!(:responder)        { create :responder, responding_teams: [bu] }
+  given!(:user_with_cases)  { find_or_create :foi_responder }
+  given!(:live_case)        { create :accepted_case }
+  given!(:responded_case)   { create :responded_case,
+                                responder: user_with_cases,
+                                responding_team: bu,
+                                received_date: 5.days.ago }
+  given!(:bu)               { find_or_create :foi_responding_team }
```

Comment:
Really small thing and I know this wasn't your choice as it existed in the code already - but could we think of a better name than `bu`? Doesn't really mean anything - I guess "business user" but I'm not sure. Should probably just use `business_user` then as it is unambiguous.

Might be worth changing while we're editing this code, like the Boy Scout's rule: https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html